### PR TITLE
updating chat.css and message.html

### PR DESCRIPTION
**Pull request by patrisiakaplun**
Adapting the design of the chat to the site template and adding functionality that allows entering the chat in the recent chats bar.

image

### CODE REVIEW

**Review by idosinger98 — APPROVED** [review]
LGTM

**Review by guybeckenstein — APPROVED** [review]
LGTM

File: chatmessage/templates/chatmessage/message.html [hunk 1]
```diff
@@ -17,7 +17,7 @@ <h4>Recent Chats</h4>
         </div>
         <div class="inbox_chat">
           {% for contact in contacts %}
-          <a href="/chatmeassage/{{ contact.profile_id.user_id.first_name }}">
+      <a href="http://127.0.0.1:8000/chatmessage/{{ user }}/{% if sender_type == 'P' %}{{ contact.professional_id }}{% else %}{{ contact.client_id }}{% endif %}/{{ sender_type }}/">
```
Comment (Yarboa) [inline chatmessage/templates/chatmessage/message.html:20]: Could not be used in production

**Review by Yarboa — CHANGES_REQUESTED** [review]

File: chatmessage/templates/chatmessage/message.html (hunk 1 above)
Comment (patrisiakaplun) [reply chatmessage/templates/chatmessage/message.html]: done.

**Review by idosinger98 — APPROVED** [review]
Lgtm

**Review by guybeckenstein — APPROVED** [review]
LGTM

**Review by Yarboa — APPROVED** [review]
LGTM

**Pull request merged**